libraries/rush-lib/package.json

```diff
@@ -57,7 +57,8 @@
     "tapable": "2.2.1",
     "tar": "~6.1.11",
     "true-case-path": "~2.2.1",
-    "uuid": "~8.3.2"
+    "uuid": "~8.3.2",
+    "pnpm-sync-lib": "0.1.0"
```

Comment:
@dmichon-msft @iclanton 

This PR is introducing a dependency on [pnpm-sync-lib](https://github.com/tiktok/pnpm-sync/tree/main/packages/pnpm-sync-lib).  Installing `rush-lib` will now install all of its dependencies:

```js
{
  "name": "pnpm-sync-lib",
  "version": "0.1.0",
  . . .
  "dependencies": {
    "@rushstack/node-core-library": "^3.61.0",
    "@rushstack/package-extractor": "^0.6.12",
    "@pnpm/lockfile-file": "~8.1.4",
    "@pnpm/logger": "~5.0.0"
  },
```

When reviewing `pnpm-sync` related pull requests, we should be aware of the source code in the other repository and its potential consequences for Rush.